django_exceptions-0.4.dist-info/INSTALLER,sha256=zuuue4knoyJ-UwPPXg8fezS7VCrXJQrAP7zeNuwvFQg,4
django_exceptions-0.4.dist-info/METADATA,sha256=zwsxWAcwi_ZmZiLMa6HWgIDlHRaIri-qeVboba17j0Q,345
django_exceptions-0.4.dist-info/RECORD,,
django_exceptions-0.4.dist-info/REQUESTED,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
django_exceptions-0.4.dist-info/WHEEL,sha256=yQN5g4mg4AybRjkgi-9yy4iQEFibGQmlz78Pik5Or-A,92
django_exceptions-0.4.dist-info/top_level.txt,sha256=4lq4T_VdpgZnf6iairB375EnWFd34ld1DPpYAj5_GCk,18
django_exceptions/__init__.py,sha256=k19_XBrRhdumMlqJ01MLWGvgvsZYZ8Q6rccfe7Ou3aY,485
django_exceptions/__pycache__/__init__.cpython-312.pyc,,
django_exceptions/__pycache__/exceptions.cpython-312.pyc,,
django_exceptions/__pycache__/middleware.cpython-312.pyc,,
django_exceptions/__pycache__/tests.cpython-312.pyc,,
django_exceptions/exceptions.py,sha256=_OOMVZq7cpV23HMV4E3avNk955aEf-yEMgUTP2O54bo,177
django_exceptions/middleware.py,sha256=X94ne_XwZSW2DfxLHg2QKxAQ2rQtJXFsE3vXk7VIdk4,931
django_exceptions/tests.py,sha256=mrbGGRNg5jwbTJtWWa7zSKdDyeB4vmgZCRc2nk6VY-g,60
